### PR TITLE
when a modal window exists, do not switch pages

### PR DESCRIPTION
**Pull request by Decodetalkers**
Log:

### CODE REVIEW

**Comment by Decodetalkers** [comment]
- https://github.com/linuxdeepin/developer-center/issues/6588

**Comment by Decodetalkers** [comment]
- https://github.com/linuxdeepin/developer-center/issues/6697

File: src/frame/mainwindow.cpp [hunk 1]
```diff
@@ -96,6 +97,10 @@ MainWindow::~MainWindow()
 
 void MainWindow::showPage(const QString &url, const UrlType &uType)
 {
+    if (DccGlobalControl::instance()->isModal()) {
```
Comment (kegechen) [inline src/frame/mainwindow.cpp:100]:
```suggestion
   if (qApp->activeModalWidget()) {
```

**Comment by zsien** [comment]
when **a** modal window exist**s**, do not switch pages (

**Comment by Decodetalkers** [comment]
> when **a** modal window exist**s**, do not switch pages (

好了

**Review by zsien — APPROVED** [review]

**Pull request merged**